Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -162,6 +163,18 @@ def custom_callback(name, progress):
 image = itk.imread(filename, imageio=itk.PNGImageIO.New())
 assert type(image) == itk.Image[itk.RGBPixel[itk.UC], 2]
 
+# Test serialization with pickle
+array = np.random.randint(0, 256, (8, 12))
```

Comment:
To keep the testing CI build quicker, not all types are wrapped, so we need to add `.astype(np.uint8)`,

```
array = np.random.randint(0, 256, (8, 12)).astype(np.uint8)
```